src/main/views/create-media-account.njk

```diff
@@ -122,8 +134,14 @@
         {{ employerInput }}
         <p class="govuk-body" tabindex="0">{{ photoMessage }}</p>
         <p class="govuk-hint" tabindex="0">{{ photoSubMessage }}</p>
-        <p class="govuk-hint" id="photoType" tabindex="0">{{ photoType }}</p>
+        <p class="govuk-hint" tabindex="0">{{ uploadConsent }}</p>
```

Comment:
Formattings slightly out on this section (just needs making in-line)

Reply:
should be fixed oto